src/test/java/org/snomed/snowstorm/rest/ReferenceSetMemberControllerTest.java

```diff
@@ -0,0 +1,82 @@
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.web.client.TestRestTemplate;
+import org.springframework.boot.web.server.LocalServerPort;
+import org.springframework.core.ParameterizedTypeReference;
+import org.springframework.http.HttpEntity;
+import org.springframework.http.HttpMethod;
+import org.springframework.test.context.ActiveProfiles;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
+
+@ActiveProfiles("test")
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT, classes = TestConfig.class)
+class ReferenceSetMemberControllerTest extends AbstractTest {
+	private static final ParameterizedTypeReference<ItemsPagePojo<ReferenceSetMember>> FIND_REFSET_MEMBERS_RESPONSE_TYPE = new ParameterizedTypeReference<>() {
+	};
+
+	@LocalServerPort
+	private int port;
+
+	@Autowired
+	private TestRestTemplate restTemplate;
+
+	@Autowired
+	private ReferenceSetMemberService referenceSetMemberService;
+
+	@Test
+	void findRefsetMembers_ShouldPage_WhenGivenSearchAfterRequestParameter() throws InterruptedException {
+		// create & assert first reference set member
+		ReferenceSetMember memberA = new ReferenceSetMember(Concepts.MODEL_MODULE, Concepts.REFSET_DESCRIPTOR_REFSET, Concepts.REFSET_MOVED_FROM_ASSOCIATION);
+		referenceSetMemberService.createMember("MAIN", memberA);
+		ItemsPagePojo<ReferenceSetMember> firstRequest = findRefsetMembers();
+		assertEquals(1, firstRequest.getItems().size()); // memberA
+
+		// create & assert second reference set member
+		ReferenceSetMember memberB = new ReferenceSetMember(Concepts.MODEL_MODULE, Concepts.REFSET_DESCRIPTOR_REFSET, Concepts.REFSET_MOVED_TO_ASSOCIATION);
+		referenceSetMemberService.createMember("MAIN", memberB);
+		assertEquals(2, findRefsetMembers().getItems().size()); // memberA & memberB
+
+		// query first page (using offset & limit)
+		ItemsPagePojo<ReferenceSetMember> firstPageMembers = findRefsetMembers(0, 1);
+		assertEquals(1, firstPageMembers.getItems().size());
+		ReferenceSetMember firstPageMember = getReferenceSetMember(firstPageMembers);
+		assertEquals(memberA.getMemberId(), firstPageMember.getMemberId());
+
+		// query second page (using searchAfter token from firstPage)
+		ItemsPagePojo<ReferenceSetMember> secondPageMembers = findRefsetMembers(firstPageMembers.getSearchAfter());
+		Thread.sleep(10000);
+		assertEquals(1, secondPageMembers.getItems().size());
+		ReferenceSetMember secondPageMember = getReferenceSetMember(secondPageMembers);
+		assertEquals(memberB.getMemberId(), secondPageMember.getMemberId());
+
+		// should be different, thus paged
+		assertNotEquals(firstPageMember, secondPageMember);
+	}
+
+	private ReferenceSetMember getReferenceSetMember(ItemsPagePojo<ReferenceSetMember> firstPageMembers) {
+		return firstPageMembers.getItems().iterator().next();
+	}
```

Comment:
Only 1 referenceSetMember is being returned at any given time (for my test) so it is safe to use iterator.